package.json

```diff
@@ -17,7 +17,7 @@
     "url": "https://github.com/NomicFoundation/hardhat-vscode/issues"
   },
```

Comment:
Does this PR close/replace #499 and #484?

Reply:
Yes, this replaces both PRs